tsconfig.build.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "downlevelIteration": true,
+    "esModuleInterop": true,
+    "lib": ["es2018"],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "sourceMap": true,
+    "strict": true,
+    "target": "es5",
+    // This check doesn't work well in monorepos: https://github.com/microsoft/TypeScript/issues/38538
+    "skipLibCheck": true,
+    "baseUrl": "./",
+    "outDir": "./dist",
+    "rootDirs": ["./contracts", "./artifacts"]
```

Comment:
I don't think you need root dirs since you generate the contracts inside the src folder.